app/codecs/CirceDecoders.scala

```diff
@@ -86,5 +87,15 @@ object CirceDecoders {
   implicit val switchStateDecode: Decoder[SwitchState] = deriveDecoder
   implicit val paymentMethodsSwitchCodec: Codec[PaymentMethodsSwitch] = deriveCodec
   implicit val switchesCodec: Codec[Switches] = deriveCodec
+  implicit val stripeApiErrorCodec: Codec[StripeApiError] = deriveCodec
+  implicit val paypalApiErrorCodec: Codec[PaypalApiError] = deriveCodec
+  implicit val decodeErrorWrapper: Decoder[ErrorWrapper] = Decoder.forProduct2("error", "type")(ErrorWrapper.apply)
+
+  implicit def decodePaymentApiError: Decoder[PaymentApiError] =
```

Comment:
For 2 decoders I think it's preferable to just use the `or` method directly:
```
Decoder[PaypalApiError].widen.or(Decoder[StripeApiError].widen)
```

Reply:
Nice!